staging/src/k8s.io/apiserver/pkg/util/flowcontrol/reqmgmt-config.go

```diff
@@ -0,0 +1,258 @@
+			reqMgmt.configQueue.AddRateLimited(obj)
+		} else {
+			reqMgmt.configQueue.Forget(obj)
+		}
+	}(obj)
+
+	return true
+}
+
+// syncOne attempts to sync all the config for the reqmgmt filter.  It
+// either succeeds and returns `true` or logs an error and returns
+// `false`.
+func (reqMgmt *requestManagementSystem) syncOne() bool {
+	all := labels.Everything()
+	newPLs, err := reqMgmt.plLister.List(all)
+	if err != nil {
+		klog.Errorf("Unable to list PriorityLevelConfiguration objects: %s", err.Error())
+		return false
+	}
+	newFSs, err := reqMgmt.fsLister.List(all)
+	if err != nil {
+		klog.Errorf("Unable to list FlowSchema objects: %s", err.Error())
+		return false
+	}
+	reqMgmt.digestConfigObjects(newPLs, newFSs)
+	return true
+}
+
+func (reqMgmt *requestManagementSystem) digestConfigObjects(newPLs []*rmtypesv1a1.PriorityLevelConfiguration, newFSs []*rmtypesv1a1.FlowSchema) {
+	oldRMState := reqMgmt.curState.Load().(*requestManagementState)
+	var shareSum float64
+	newRMState := &requestManagementState{
+		priorityLevelStates: make(map[string]*priorityLevelState),
+	}
+	newlyQuiescent := make([]*priorityLevelState, 0)
+	for _, pl := range newPLs {
+		state := oldRMState.priorityLevelStates[pl.Name]
+		if state == nil {
+			state = &priorityLevelState{
+				config: pl.Spec,
+			}
+		} else {
+			if state.config.Exempt {
+				newRMState.priorityLevelStates[pl.Name] = state
+				continue
+			}
+
+			oState := *state
+			state = &oState
+			state.config = pl.Spec
+			if state.emptyHandler != nil { // it was undesired, but no longer
+				klog.V(3).Infof("Priority level %s was undesired and has become desired again", pl.Name)
+				state.emptyHandler = nil
+				state.queues.Quiesce(nil)
+			}
+		}
+		shareSum += float64(state.config.AssuredConcurrencyShares)
+		newRMState.priorityLevelStates[pl.Name] = state
+	}
+
+	fsSeq := make(rmtypesv1a1.FlowSchemaSequence, 0, len(newFSs))
+	for _, fs := range newFSs {
+		reqMgmt.syncFlowSchemaStatus(fs, newRMState.priorityLevelStates[fs.Spec.PriorityLevelConfiguration.Name] == nil)
+	}
+	sort.Sort(fsSeq)
+
+	if klog.V(5) {
+		for _, fs := range fsSeq {
+			klog.Infof("Using FlowSchema %s: %#+v", fs.Name, fs.Spec)
+		}
+	}
+	for plName, plState := range oldRMState.priorityLevelStates {
+		if newRMState.priorityLevelStates[plName] != nil {
+			// Still desired
+		} else if plState.emptyHandler != nil && plState.emptyHandler.IsEmpty() {
+			// undesired, empty, and never going to get another request
+			klog.V(3).Infof("Priority level %s removed from implementation", plName)
+		} else if !plState.config.Exempt {
+			oState := *plState
+			plState = &oState
+			if plState.emptyHandler == nil {
+				klog.V(3).Infof("Priority level %s became undesired", plName)
+				plState.emptyHandler = &emptyRelay{reqMgmt: reqMgmt}
+				newlyQuiescent = append(newlyQuiescent, plState)
+			}
+			newRMState.priorityLevelStates[plName] = plState
+			shareSum += float64(plState.config.AssuredConcurrencyShares)
+		}
```

Comment:
How about something like
```
else {
    klog.V(3).Info(`Ignoring deletion of PriorityLevelConfiguration "exempt"`)
}
```

Reply:
This has been partially handled now.  The missing part is including this priority's contributio to `shareSum` in the non-exempt case.